### DISCUSSION TITLE
Czech koruna

### DISCUSSION THREAD

**Discussion by nexovec** [firefly-iii/firefly-iii#8290] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

Please add "Czech koruna" to currencies.

### Solution

I have no idea. Without support for my local currency I can't really evaluate the program. Looks awesome so far though.

**Comment by JC5** [comment]
You can add it yourself under /currencies. Good luck!
